_includes/releases/v22.2/v22.2.0-beta.5.md

```diff
@@ -0,0 +1,70 @@
+## v22.2.0-beta.5
+
+Release Date: October 31, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-5-general-changes">General changes</h3>
+
+- Cockroach Docker images are now multi-architecture manifests, supporting both the `x86_64` (`amd64`) and `arm64` architectures. [#90304][#90304]
```

Comment:
I'd move this near line 44 rather than in a category all on its own. Also I'd say "Docker images for CockroachDB".